specification/managedservices/resource-manager/Microsoft.ManagedServices/preview/2020-02-01-preview/managedservices.json

```diff
@@ -887,21 +887,21 @@
       "properties": {
         "managedByTenantId": {
           "type": "string",
-          "description": "Id of the managedBy tenant."
+          "description": "The identifier of the managedBy tenant."
         },
         "authorizations": {
           "type": "array",
           "items": {
             "$ref": "#/definitions/Authorization"
           },
-          "description": "Authorization tuple containing principal id of the user/security group or service principal and id of the build-in role."
+          "description": "The collection of principals in the managedBy tenant with permanent access to the managed tenant."
```

Comment:
>The collection of principals in the managedBy tenant with permanent access to the managed tenant. [](http://example.com/codeflow?start=26&length=97)

address it as per previous comment.

Reply:
also these are not only just collection of principles.

Reply:
Should be fixed now w/ new wording